src/Parts/AllTrainingEventTable/index.jsx

```diff
@@ -0,0 +1,63 @@
+import LabelSection from '../../Components/LabelSection'
+import TableData from '../../Components/TableData'
+import Data from '../../Dummy/DataTable.json'
+import { Rate } from 'antd'
+function AllTrainingEventTable () {
```

Comment:
Please change with arrow function.
*based on pattern

Reply:
thankyou for your review, this issue fix in 327bdc89e17135a0db052aba8aa4120c69ceb0f2